hubblestack/utils/event.py

```diff
@@ -0,0 +1,851 @@
+# -*- coding: utf-8 -*-
+'''
+Manage events
+
+Events are all fired off via a zeromq 'pub' socket, and listened to with local
```

Comment:
I believe, we dont need this file either. We don't use events (zeromq)

Reply:
Events is needed in roots fileserver